tests/manage/mcg/test_bucket_deletion.py

```diff
@@ -21,11 +21,13 @@
 )
 from ocs_ci.helpers.helpers import create_unique_resource_name
 from ocs_ci.framework.testlib import MCGTest
+from ocs_ci.framework.pytest_customization.marks import skipif_openshift_dedicated
 
 logger = logging.getLogger(__name__)
 ERRATIC_TIMEOUTS_SKIP_REASON = "Skipped because of erratic timeouts"
 
 
+@skipif_openshift_dedicated
```

Comment:
Do we want to see this test case in skipped in each of execution run on OSD?

Other approach is to deselect this test case from collection and this case it will not be visible in collection and summary at all?

If we don't want to even collect this test case for OSD and not be part of skips in the execution I suggest to use different approach like we do here:

https://github.com/red-hat-storage/ocs-ci/blob/master/tests/conftest.py#L113

Which will deselect the test case from collection and it will not be visible.

@ebenahar @RazTamir @vasukulkarni  FYI

Reply:
+1 Petr on deselcting instead of skipping. 
We can follow the same approach as done for RGW tests https://github.com/red-hat-storage/ocs-ci/blob/5b41b78a63433ac2ae8437a791a8a3644322a624/tests/manage/rgw/conftest.py#L8 on non on-prem platforms

Reply:
@petr-balogh @ebenahar This patch is a temporary solution to skip the mcg related tests, in future openshift dedicated may support for mcg and its okay to see them skipped in the test runs.

Reply:
+1 to Petr suggestion.
Skips for such reasons are just polluting the reports